core/src/test/java/org/infinispan/tx/dld/ControlledRpcManager.java

```diff
@@ -125,11 +122,11 @@ protected Object beforeInvokeRemotely(ReplicableCommand command) {
    }
 
    @Override
-   protected Map<Address, Response> afterInvokeRemotely(ReplicableCommand command, Map<Address, Response> responseMap, Object argument) {
+   protected Object afterInvokeRemotely(ReplicableCommand command, Object responseObject, Object argument) {
```

Comment:
I thought it would make it harder for implementers, but it looks about the same so I'm changing it.